src/attrib/workarounds.ts

```diff
@@ -148,6 +148,140 @@ workaround("Make English Wynguard Forces available from Imperial Age", {
   ...overrideAge(["wynguard-army", "wynguard-footmen", "wynguard-raiders", "wynguard-rangers"], 4, ["en"]),
 });
 
+//could do this by creating dependency from techs that unlock abilities, but doing workaround for the time being
+
+workaround("Make Ability Arrow Volley availabe in Imperial Age", {
+  ...overrideAge(["arrow-volley"], 4, ["en"]),
+});
+
+workaround("Make Ability Saints Blessing availabe in Castle Age", {
+  ...overrideAge(["saints-blessing"], 3, ["ru"]),
+});
+
+workaround("Make Ability High Armory Production Bonus availabe in Imperial Age", {
+  ...overrideAge(["high-armory-production-bonus"], 4, ["ru"]),
+});
+
+workaround("Make Ability Static Deployment availabe in Imperial Age", {
+  ...overrideAge(["static-deployment"], 4, ["ru"]),
+});
+
+workaround("Make Ability Gallop availabe in Imperial Age", {
+  ...overrideAge(["gallop"], 4, ["ru"]),
+});
+
+workaround("Make Ability Gallop availabe in Imperial Age", {
+  ...overrideAge(["gallop"], 4, ["ru"]),
+});
+
+workaround("Make Ability Fortitude availabe in Feudal Age", {
+  ...overrideAge(["fortitude"], 2, ["ot"]),
+});
+
+workaround("Make Ability blacksmith-and-university-influence availabe in Feudal Age", {
+  ...overrideAge(["blacksmith-and-university-influence"], 2, ["ot"]),
+});
+
+
+
+// ---- Abilities ----
+
+workaround("Fix age and add ability props for great_wall_buff", {
+  predicate: (item) => item.type === "ability" && item.attribName === "great_wall_buff_chi",
+  mutator: (item) => {
+    item.age = 4;
+    item.id = `${item.baseId}-${item.age}`;
+    item.activation = "always";
+    item.range = 0;
+    item.costs = { ...NO_COSTS, popcap:0 };
+  },
+});
+
+workaround("Fix age and add missing info for spirit_way", {
```

Comment:
It's pretty annoying the info in files like `attrib/instances/info/buff_info/races/chinese/spirit_way_chi.xml` isn't logically referenced from `abilities/always_on_abilities/chinese/spirit_way.xml` or vice versa.

Roughly looking at all the buffs I would say that maybe we can get away with auto joining in certain buff info based on (partial) name match? Or otherwise some additional configuration/mapping near the hardcoded ones. It seems the dev have some hardcoded things going in themselves. 

Especially hardcoded translations like the one below are high maintenance otherwise.

Reply:
There are so few info/buff_infos we resort to that I think the static method we have suffices.